cmd/secrets-store-csi-driver/main.go

```diff
@@ -64,7 +64,8 @@ var (
 
 	// enable filtered watch for NodePublishSecretRef secrets. The filtering is done on the csi driver label: secrets-store.csi.k8s.io/used=true
 	// For Kubernetes secrets used to provide credentials for use with the CSI driver, set the label by running: kubectl label secret secrets-store-creds secrets-store.csi.k8s.io/used=true
-	filteredWatchSecret = flag.Bool("filtered-watch-secret", true, "enable filtered watch for NodePublishSecretRef secrets with label secrets-store.csi.k8s.io/used=true")
+	// This feature is enabled by default starting v0.1.0 and can't be disabled starting v1.0.0 release.
+	_ = flag.Bool("filtered-watch-secret", true, "enable filtered watch for NodePublishSecretRef secrets with label secrets-store.csi.k8s.io/used=true")
```

Comment:
should we add a `klog.Error` or warning if this is set to `false` to tell users that it has no effect?

Reply:
Added a warning message: `Filtered watch for nodePublishSecretRef secret based on secrets-store.csi.k8s.io/used=true label can't be disabled. The --filtered-watch-secret flag will be deprecated in future releases.`. Lmk what you think.